### ISSUE TITLE
Downloads panel isn't rendered, since not existing method is referenced

### ISSUE THREAD

**Issue by OleksandrBarybin** [adobe/asset-share-commons#1095]
**Describe the bug**
https://github.com/adobe/asset-share-commons/blob/a742a0adfeef77e969698fb7ee900fd042fd45a2/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/structure/user-menu/user-menu.html#L39 using a method from https://github.com/adobe/asset-share-commons/blob/develop/core/src/main/java/com/adobe/aem/commons/assetshare/configuration/Config.java that doesn't exist, so this section is never rendered.

Looks like https://github.com/adobe/asset-share-commons/blob/a742a0adfeef77e969698fb7ee900fd042fd45a2/core/src/main/java/com/adobe/aem/commons/assetshare/configuration/Config.java#L116 should be used instead?

**Environment**
* AEM Version: 6.5
* Asset Share Commons Version: 3.8.2
* Author, Publish or both? both

**To Reproduce**
1. Go to Search page
2. Download section is never rendered, even if download is enabled.

**Expected behavior**
Proper method from https://github.com/adobe/asset-share-commons/blob/a742a0adfeef77e969698fb7ee900fd042fd45a2/core/src/main/java/com/adobe/aem/commons/assetshare/configuration/Config.java should be used

**Additional context**
Was also observed here #1013

**Comment by dragandimic** [comment]
Bug Confirmed.

**Comment by davidjgonzalez** [comment]
Im gonna have to test this out, but i think if we change `user-menu.html` to have this HTL, it should do it.

```
  <!--/* Downloads - Only show on AEM as a Cloud Service when downloads are enabled */-->
<a class="item" data-sly-test="${config.downloadEnabled && !config.aemClassic}"
    data-asset-share-id="show-downloadspanel"> <i
    class="big download icon"></i>
    <div class="floating ui primary circular label"
        data-asset-share-id="downloads-count"></div>
</a>
```

**Comment by davidjgonzalez** [comment]
Oh - nevermind .. strange.... its in both `header.html` and `user-menu.html`   ... this hasnt been a "problem" because as you note, the one in `user-menu.html` will never show since the method is not implemented.

![2024-03-13 at 4 27 PM](https://github.com/adobe/asset-share-commons/assets/1451868/10709ecb-4f66-46fd-a601-fdbdb3e0088e)

**Comment by davidjgonzalez** [comment]
Ok - so this is what Desktop should look like:

![2024-03-13 at 4 34 PM](https://github.com/adobe/asset-share-commons/assets/1451868/422f6450-ffe5-49e0-b383-087c7f6cf1c2)


and this is what mobile should look like:

![2024-03-13 at 4 34 PM](https://github.com/adobe/asset-share-commons/assets/1451868/8207c599-f5a8-4243-bcbe-92a15defbc7b)

**Comment by davidjgonzalez** [comment]
Fixed in: https://github.com/adobe/asset-share-commons/pull/1096

**Comment by davidjgonzalez** [comment]
Fixed in 3.8.4

**Comment by dragandimic** [comment]
Awesome, thank you for the quick fix.
